.env.example

```
@@ -1,40 +1,209 @@
-GRAFANA_PASSWORD="VaalEmpire2025!"
-
-# Vercel Deployment
-VERCEL_TOKEN=your-vercel-token
-VERCEL_ORG_ID=your-org-id
-VERCEL_PROJECT_ID=your-project-id
-HUBSPOT_PRIVATE_APP_TOKEN='your-hubspot-private-app-token'
+# This is an example .env file for local development.
+# For production, use your cloud provider's secret management service.
+
+# Core Application
+NODE_ENV="development"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)